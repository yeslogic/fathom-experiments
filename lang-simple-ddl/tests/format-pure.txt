format test1 :=
  pure (A := Int64) 3;

format test2 :=
  pure (3 : Int64);

format test3 :=
  let x : Int64 := 3;
  pure x;
