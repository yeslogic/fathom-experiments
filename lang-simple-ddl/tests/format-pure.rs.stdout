fn read_test2(input: &[u8], pos: &mut usize) -> Result<i64, ()> {
    Ok(3)
}

fn read_test1(input: &[u8], pos: &mut usize) -> Result<i64, ()> {
    Ok(3)
}

fn read_test3(input: &[u8], pos: &mut usize) -> Result<i64, ()> {
    let x: i64 = 3;
    Ok(x)
}
